MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -313,9 +313,13 @@ private void calcMedicalSupportHoursNeeded() {
         int leftOver = activePersonnelCount - (numSquads * 7);
 
         medSupportNeeded = (numSquads * 4) +
-                           (3 + (new BigDecimal(leftOver).divide(
+                           (new BigDecimal(leftOver).divide(
```

Comment:
This cancels out the contribution of left over units below 5 personnel.

Re-reading the section in FM:Mr(Rev) and following the examples provided, the following logic should work:
```
// Calculated based on 7-person squads
int numSquads = (int)(activePersonnelCount / 7); // integer division intended
// each squad takes 4.4 hours (3 + 7/5) rounded to the nearest hour
int medSupportNeeded = numSquads * 4;
int leftOver = activePersonnelCount - (numSquads * 7);
if (leftOver > 0) {
    // the left overs form a squad which needs (3 + N / 5) rounded hours
    medSupportNeeded += (int)Math.round(3 + leftOver / 5.0);
}
```

Reply:
Examples from FM:Mr(Rev):
> 1 infantry platoon (Total tonnage: 4)
> Support needs: 16 man-hours 

> 36 MechWarriors, Pilots, DropShip crew, and non-combatants
> Support needs: 23 man-hours (medical)